interfaces/portal/src/app/pages/project-settings-fsps/components/fsp-configuration-card/fsp-configuration-card.component.ts

```diff
@@ -0,0 +1,135 @@
+import { ToastService } from '~/services/toast.service';
+import { TranslatableStringService } from '~/services/translatable-string.service';
+
+@Component({
+  selector: 'app-fsp-configuration-card',
+  imports: [
+    TranslatableStringPipe,
+    ButtonModule,
+    FormDialogComponent,
+    TableModule,
+    CardWithLinkComponent,
+    EllipsisMenuComponent,
+  ],
+  templateUrl: './fsp-configuration-card.component.html',
+  styles: ``,
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  providers: [ToastService],
+})
+export class FspConfigurationCardComponent {
+  readonly projectId = input.required<string>();
+  readonly configuration = input.required<FspConfiguration>();
+  readonly reconfigureFsp = output<FspConfiguration>();
+
+  fspConfigurationService = inject(FspConfigurationService);
+  fspConfigurationApiService = inject(FspConfigurationApiService);
+  projectApiService = inject(ProjectApiService);
+  translatableStringService = inject(TranslatableStringService);
+  toastService = inject(ToastService);
+
+  projectAttributes = injectQuery(
+    this.projectApiService.getProjectAttributes({
+      projectId: this.projectId,
+      includeProgramRegistrationAttributes: true,
+    }),
+  );
+
+  readonly deleteConfirmationDialog = viewChild.required<FormDialogComponent>(
+    'deleteConfigurationDialog',
+  );
+
+  readonly deleteConfigurationDialogHeader = computed(
+    () =>
+      $localize`Remove` +
+      ` "${this.translatableStringService.translate(this.configuration().label) ?? ''}"`,
+  );
+
+  readonly fspSetting = computed(
+    () => FSP_SETTINGS[this.configuration().fspName],
+  );
+
+  readonly fspImage = computed(
+    () => FSP_IMAGE_URLS[this.configuration().fspName],
+  );
+
+  readonly menuItems = computed<MenuItem[]>(() => [
+    {
+      label: 'Reconfigure',
+      icon: 'pi pi-pencil',
+      command: () => {
+        this.reconfigureFsp.emit(this.configuration());
+      },
+    },
+    {
+      label: 'Delete',
+      icon: 'pi pi-trash',
+      command: () => {
+        this.deleteConfirmationDialog().show();
+      },
+    },
+  ]);
+
+  readonly requiredRegistrationAttributes = computed(() => {
+    const requiredFspAttributes =
+      this.fspConfigurationService.getRequiredFspAttributes({
+        fspSetting: this.fspSetting(),
+        existingFspConfiguration: this.configuration(),
+      });
+
+    return requiredFspAttributes.map((propertyName) =>
+      this.projectAttributes.data()?.find((attr) => attr.name === propertyName),
+    );
+  });
+
+  deleteConfigurationMutation = injectMutation(() => ({
+    mutationFn: () =>
+      this.fspConfigurationApiService.deleteFspConfiguration({
+        projectId: this.projectId,
+        configurationName: this.configuration().name,
+      }),
+    onSuccess: () => {
+      void this.projectApiService.invalidateCache(this.projectId);
+      void this.fspConfigurationApiService.invalidateCache(this.projectId);
```

Comment:
This step would be possible to do "in" the `fspConfigurationApiService.deleteFspConfiguration`-method, right?

Partly that would be more convenient (1 thing less to forget); But you would still need to manually invalidate the program/project-cache :think:

Nevermind... we'll address the caching-thing in the generic way anyway soon...